cfme/tests/infrastructure/test_customization_template.py

```diff
@@ -99,3 +99,29 @@ def test_name_max_character_validation(collection):
             script_type='Kickstart',
             script_data='Testing the script')
     collection.delete(False, template_name)
+
+
+def test_customization_template_copy(collection):
+    template_crud = collection.create(name=fauxfactory.gen_alphanumeric(8),
+                                      description=fauxfactory.gen_alphanumeric(16),
+                                      image_type='RHEL-6',
+                                      script_type='Kickstart',
+                                      script_data='Testing the script')
+    copy_template_crud = collection.copy(template_crud)
+    collection.delete(False, copy_template_crud, template_crud)
+
+
+def test_customization_template_copy_validation(appliance, collection):
+    template_crud = collection.create(name=fauxfactory.gen_alphanumeric(8),
```

Comment:
REQUIRED: add small doctrings explaining the purpose/intent of the test. that's a musthave right not for new tests